### ISSUE TITLE
[1.21] Functional Storage Access Points are losing their ability to connect to drawer controller

### ISSUE THREAD

**Issue by yepbeeble** [Buuz135/FunctionalStorage#327]
**DO NOT REPORT AN ISSUE IF YOU HAVEN'T TESTED THE ISSUE WITH THE LATEST VERSION IN CURSEFORGE**

Functional Storage Version 1.3.3

Titanium Version 4.0.21

Randomly, as well as on world load my functional storage access points keep disconnecting from pipes, becoming unavailable to import to a drawer connected to the system. 
Case 1
I have an Extended Industrialization Processing Array using Pipez universal cable with an Infinity Upgrade pumping various liquids into a drawer controller access point. Randomly, and with specific items/fluids it will stop accepting items through these pipes. Replacing the controller and re-linking to the network does not fix the problem. Breaking the pipe and replacing it fixes the issue. I still believe this to be a problem with the access point, because of Case 2.
Case 2
I had another Extended Industrialization processing array connected to a storage controller access point, and since I was troubleshooting whether pipez was the problem, I switched to using a modular router to pull from the output of the processing array, and pushing into the drawer controller access point, where it worked temporarily. Then shortly after, it stopped accepting items. The buffer of the modular router filled, as well as the processing array. Re-defining the sender module in the router to the access point WITHOUT breaking and replacing it fixed the issue. 

tl;dr randomly, access points stop accepting items/fluids.

Current Behavior (bugged)
![image](https://github.com/user-attachments/assets/bdfe1c0c-e99e-4092-ac72-494226285016)
![image](https://github.com/user-attachments/assets/7e21a310-a30a-42ad-b0ab-804e8e7352f8)

Expected Behavior
Re-placed mekanism pipe AND re-defined sender modules destination. Did NOT break Access Point.

![image](https://github.com/user-attachments/assets/b5a45050-0756-46b4-83db-1ebb8d580037)
![image](https://github.com/user-attachments/assets/cee50edc-373b-4318-8303-d5acee3a6037)
